### DISCUSSION TITLE
RIP Bruce_Browningg. (More Advice)

### DISCUSSION THREAD

**Discussion by Bruce-Browninggg** [aeolus-1/triangleGame.github.io#51] [Ideas]
Loading screen looks good with the exception of some of the "tips" which just ruin the authenticity of the game. I have a few pieces of advice.

1) Do not include personal clues, they take the game from an authentic game to being bad. 
2) This includes the questionnaire, create a more generic question 
3) Create some credits, credit the developer and anyone that helped 
4) Add clues that hint at the hidden areas, because there are no clues no-one will find the extra parts

Please follow this advice, i believe this could really improve the game.

**Comment by aeolus-1** [comment]
no
